examples/dashboard-operator/config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/jrjohnson-gke/dashboard-operator:latest
+      - image: /dashboard-operator:latest
```

Comment:
If `kustomize build` generate manifest from this, `image` is set to `/dashboard-operator:latest` without above comment( it say "Change the value of image ..." ).
IMHO, it is better that add some prefix saying "change me" as value of 'image'.
e.g. `<my-cool-project>/dashboard-operator:latest` or `<change-me>/dashboard-operator:latest` or something else.

Reply:
Oh, it's merged, sorry for late comment.

Reply:
This should be filled in by the kubebuilder generated: https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/blob/master/examples/dashboard-operator/Makefile#L69 and ultimately consumed by kustomize. 

I agree an invalid image name here would be a better default than just the `/..`